docker-compose.yaml

```diff
@@ -48,18 +48,21 @@ services:
 
   redis:
     image: redis:7-alpine
+    ports:
+      - 6379:6379
 
   celery-worker:
     build:
       context: .
       dockerfile: ./compose/celery/Dockerfile
     env_file:
       - .env.shared
+    environment:
+      - C_FORCE_ROOT=true
```

Comment:
I'm using pickle to serialize the ComponentObject. Celery recommends not doing it, because it's not 100% secure. The alternative is to serialize with JSON. Therefore we have to implement serialization and deserialization for every component. Because that would take some time, I decided to use the unsecured and fast way for the MVP. 
To do that, we need to enable `C_FORCE_ROOT`.